### PR TITLE
[ERSSUP-54586]-[]-[APIM User calls e-RS Endpoint to Retrieve Appointment Request Worklist]-[HR]

### PR DESCRIPTION
**Pull request by nhsd-hubert-raich**
## Summary
* Routine Change
* :exclamation: Breaking Change
* :robot: Operational or Infrastructure Change
* :sparkles: New Feature
* :warning: Potential issues that might be caused by this change

Add any other relevant notes or explanations here. **Remove this line if you have nothing to add.**

## Reviews Required
* [x] Dev
* [ ] Test
* [ ] Tech Author
* [ ] Product Owner

## Review Checklist
:information_source: This section is to be filled in by the **reviewer**.

* [ ] I have reviewed the changes in this PR and they fill all or part of the acceptance criteria of the ticket, and the code is in a mergeable state.
* [ ] If there were infrastructure, operational, or build changes, I have made sure there is sufficient evidence that the changes will work.
* [ ] I have ensured the changelog has been updated by the submitter, if necessary.

### CODE REVIEW

**Review by adamoldfield-nhs — APPROVED** [review]
I've reviewed the generated documentation and this looks good.

I haven't done a detailed review of this pull request, I'll delegate to @nhsd-jack-wainwright for that.

**Review by nhsd-jack-wainwright — COMMENTED** [review]
Had a look both at the file changes in the review and the preview on the UAT Bloomreach instance. Looks good just a few minor comments :).

File: specification/components/schemas/eRS-OperationOutcome.yaml [hunk 1]
```diff
@@ -29,7 +29,6 @@ properties:
       required:
         - severity
         - code
-        - diagnostics
```
[the remark this answers is not in the document]
Comment (nhsd-hubert-raich) [reply specification/components/schemas/eRS-OperationOutcome.yaml:32]: In the errors within fetchWorklist flow. I was surprised by it too

File: sandbox/src/mocks/retrieveWorklist/responses/ReferralsForReview.json [hunk 2]
```diff
@@ -0,0 +1,223 @@
...
+                    "system": "_baseUrl_/STU3/CodeSystem/SPECIALTY",
+                    "code": "CARDIOLOGY"
+                  }
+                ]
+              }
+            },
+            {
+              "url": "patient",
+              "valueReference": {
+                "reference": "#Patient-9462979626"
```
[the remark this answers is not in the document]
Comment (nhsd-hubert-raich) [reply sandbox/src/mocks/retrieveWorklist/responses/ReferralsForReview.json:87]: I think it matches, we have 2 contained patient resources there. 
-Patient-9462979626
-Patient-1000000001

File: specification/components/schemas/eRS-FetchWorklist-List.yaml [hunk 3]
```diff
@@ -0,0 +1,722 @@
...
+          enum:
+            - 'https://fhir.nhs.uk/STU3/StructureDefinition/eRS-FetchWorklist-List-1'
+          example: 'https://fhir.nhs.uk/STU3/StructureDefinition/eRS-FetchWorklist-List-1'
+  contained:
+    type: array
+    description: |
+      | Field                  | Cardinality | Notes                                      |
+      | ---------------------- | ----------- | ------------------------------------------ |
+      | `Practitioner`         | `0..*`      |                                            |
+      | `Patient`              | `0..*`      |                                            |
```
[the remark this answers is not in the document]
Comment (nhsd-hubert-raich) [reply specification/components/schemas/eRS-FetchWorklist-List.yaml:30]: Done:)

File: specification/components/schemas/eRS-FetchWorklist-List.yaml [hunk 4]
```diff
@@ -0,0 +1,722 @@
...
+                                - patient
+                              example: 'patient'
+                            valueReference:
+                              type: object
+                              required:
+                                - reference
+                              properties:
+                                reference:
+                                  type: string
+                                  example: '#Patient-9462979626'
```
[the remark this answers is not in the document]
Comment (nhsd-hubert-raich) [reply specification/components/schemas/eRS-FetchWorklist-List.yaml:242]: This line is specifically applying for Extension-eRS-ReferralsforReview-WorkListItem type. There is another patient definition defined at line 517 that describes what the patient schema is for Extension-eRS-AppointmentSlotIssues-WorkListItem. Hopefully that clears it up :)

File: specification/components/schemas/eRS-OperationOutcome.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (nhsd-hubert-raich) [reply specification/components/schemas/eRS-OperationOutcome.yaml:32]: Checked the backend code and an example where this comes from is on RetrieveWorklistRequestValidator line 115:
	ValidationUtil.throwApiValidationException(BusinessOutcomeKey.NO_RELATIONSHIP);

**Review by nhsd-nigel-mellor — APPROVED** [review]

**Pull request merged**